src/main/java/ch/njol/skript/conditions/CondIsHandRaised.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.EquipmentSlot;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Is Hand Raised")
+@Description("Checks whether an entity has one or both of their hands raised. " +
+		"Hands are raised when an entity is using an item (eg: blocking, drawing a bow, eating).")
+@Examples({"on damage of player:",
+	"\tif victim's main hand is raised:",
+	"\t\tdrop player's tool at player",
+	"\t\tset player's tool to air"})
+@Since("INSERTVERSION")
+@RequiredPlugins("Paper 1.12.2 or newer")
+public class CondIsHandRaised extends Condition {
```

Comment:
Not easily. Grammatically, it shouldn't be ``player is main hand raised``, it's ``player's main hand is raised``
So unless ``player's main hand`` gets its own type, I don't think it can be, unless I'm mis-interpreting something.

Reply:
You could make the class itself a `PropertyCondition` but use the regular `Skript.registerCondition` method for registering the condition

Reply:
ahhh i got it, good idea